lib/tasks/taxonomy/bulk_tag_document_type.rake

```diff
@@ -7,11 +7,11 @@ namespace :taxonomy do
     A JSON representation of the output is sent to STDOUT and can be
     redirected to a file if needed.
   DESC
-  task :bulk_tag_document_type, %i[document_type taxon_base_path] => :environment do |_, args|
+  task :bulk_tag_document_type, %i[document_type taxon_content_id] => :environment do |_, args|
```

Comment:
I'm not sure where this rake task is used. Do we need to change the rake task parameters in a job somewhere or is this a developer task to manually run this rake task?

Reply:
This is something a developer does. We are not expecting this type of bulk tagging to happen a lot (if at all) after doing the list on the ticket, so we decided it should be a rake task instead of cluttering up the Content tagger UI